config/permissions.php

```diff
@@ -81,7 +81,17 @@
     'manager' => array(
         '/projectsEvaluation.php', '/usersEvaluation.php', '/projectsSummary.php', '/services/getFilteredCustomProjectsService.php',
         '/services/getUserStoryReportJsonService.php', '/services/getProjectCustomerReportJsonService.php',
-        '/services/getProjectUserReportJsonService.php',
+        '/services/getProjectUserReportJsonService.php', '/services/getCustomerProjectsService.php',
+        //reports
+        '/viewProjectDetails.php', '/viewUserDetails.php', '/projectDetails.php', '/viewWorkingHoursResultsReport.php',
+        '/services/getExtraHoursReportService.php','/services/getPendingHolidayHoursService.php',
+        '/services/getProjectTtypeReportService.php', '/services/getProjectUserCustomerReportJsonService.php',
+        '/services/getProjectUserCustomerReportService.php', '/services/getProjectUserStoryReportService.php',
+        '/services/getUserProjectCustomerReportJsonService.php', '/services/getUsersProjectsReportService.php',
+        '/services/getProjectUserStoryReportJsonService.php',
+        '/services/getProjectUserWeeklyHoursReportJsonService.php', '/services/getUserCustomersService.php',
+        '/services/getUserProjectCustomerReportJsonService.php'
+
```

Comment:
As things are now, an admin must be also member of staff for things to work correctly. It's reasonable to consider that a manager should also be staff, so you don't have to repeat the full list of report pages here.

So it was correct before the change :)